src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -1,15 +1,32 @@
+/*
+ * File: z_bg_mizu_movebg.c
+ * Overlay: ovl_Bg_Mizu_Movebg
+ * Description: Kakariko Village Well Water
+ */
+
 #include "z_bg_mizu_movebg.h"
+#include "overlays/actors/ovl_Bg_Mizu_Water/z_bg_mizu_water.h"
+#include "objects/object_mizu_objects/object_mizu_objects.h"
 
 #define FLAGS 0x00000010
 
 #define THIS ((BgMizuMovebg*)thisx)
 
+#define BgMizuMovebgGetType(params) (((u16)(params) >> 0xC) & 0xF)
+#define BgMizuMovebgGetFlags(params) ((u16)(params)&0x3F)
+#define BgMizuMovebgGetPathId(params) (((u16)(params) >> 0x8) & 0xF)
+#define BgMizuMovebgGetPointId(params) ((u16)(params)&0xF)
+#define BgMizuMovebgGetSpeed(params) (((u16)(params) >> 0x4) & 0xF)
```

Comment:
we arent doing macros for these until we agree on a standardize way to do them. it is planned for a documentation pass, which will be really soon as we are almost done with actors